src/js/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ErrorPage = ({ location }) => {
+  const { state } = location;
+
+  // Check if the pages was redirected with an error
+  const error = state ? state.error : '404';
+
+  return (
+    <div className="landing-page">
+      <div className="landing-page-content">
+        <div className="error-text">{`Error: ${error}`}</div>
```

Comment:
In a real-world situation, we would want the browser to respond with a `404` status code in the HTTP response. Not sure how to do that with React Router, but I thought it was worth mentioning.